jupyterhub/log.py

```diff
@@ -120,3 +121,4 @@ def log_request(handler):
         if location:
             ns['location'] = ' → {}'.format(location)
     log_method(msg.format(**ns))
+    prometheus_log_method(handler)
```

Comment:
Add to docstring: `- log prometheus metrics`

Reply:
Done!